src/ui/Pricing/Comparison/comapre.ts

```diff
@@ -190,5 +190,5 @@ export const PRO_PLUS_COMPARED_FEATURES = {}
 export const PlanFeatures = {
   [Plan.FREE]: FREE_COMPARED_FEATURES,
   [Plan.PRO]: PRO_COMPARED_FEATURES,
-  [Plan.PRO_PLUS]: PRO_PLUS_COMPARED_FEATURES,
```

Comment:
`[Plan.PRO_PLUS]` should not be removed to ensure compatibility with legacy systems